gnovm/stdlibs/std/banker.gno

```diff
@@ -62,6 +68,23 @@ func (ba bankAdapter) TotalCoin(denom string) int64 {
 }
 
 func (ba bankAdapter) IssueCoin(addr Address, denom string, amount int64) {
+	// Similar to ibc ==> "ibc/" + sha256('path' + 'base_denom')
+	// "realm/" + sha256('pkg_path' + '/' +  'base_denom')
```

Comment:
```suggestion
	// "realm/" + 'pkg_path' + '/' +  'base_denom'
```

Not sure if possible yet, but would be nice to have this instead

Reply:
Since denom is `string`, it might possible.
However is it good looking string to have multiple `/` in denom?

```
pkg_path = "gno.land/r/outer1/inner1"
denom = "gnot"
```
final denom will be `realm/gno.land/r/outer1/inner1/gnot`

Reply:
LGTM :)

Maybe we can keep sha256 `if len(pkg_path) + len(base_denom) > upperLimit`

while having support for readable denoms for most cases

Reply:
Sounds great :D. I think length of 30 ~ 40 is good for limit.